canvas_account_admin_tools/settings/base.py

```diff
@@ -263,9 +263,17 @@
 ICOMMONS_REST_API_TOKEN = SECURE_SETTINGS.get('icommons_rest_api_token')
 ICOMMONS_REST_API_SKIP_CERT_VERIFICATION = False
 
-PERMISSION_ACCOUNT_ADMIN_TOOLS = 'account_admin_tools'
-PERMISSION_PEOPLE_TOOL = 'people_tool'
-PERMISSION_XLIST_TOOL = 'cross_listing'
+# TOOL_PERMISSIONS is used to map app names to SchoolPermission.permission names
+# to determine visibility of tools on dashboard and access to tool views/pages
+TOOL_PERMISSIONS = {
+    # NOTE: canvas_account_admin_tools permission name is shortened to
+    # account_admin_tools due to SchoolPermission.permission field max length
```

Comment:
I am good with using list here and also increasing the column width  in the SchoolPermission table to be beyond 25. Now maybe a good time to do it since we are just creating it?